src/serverless_monitors.ts

```diff
@@ -0,0 +1,73 @@
+export interface ServerlessMonitor {
+  name: string;
+  query: (cloudFormationStackId: string) => string;
+  message: string;
+  type?: string;
+}
+
+export const HIGH_ERROR_RATE: ServerlessMonitor = {
+  name: "High Error Rate on {{functionname.name}} in {{region.name}} for {{aws_account.name}}",
+  query: (cloudFormationStackId: string) => {
+    return `avg(last_15m):sum:aws.lambda.errors{aws_cloudformation_stack-id:${cloudFormationStackId}} by {functionname,region,aws_account}.as_count() / sum:aws.lambda.invocations{aws_cloudformation_stack-id:${cloudFormationStackId}} by {functionname,region,aws_account}.as_count() >= 0.1`;
+  },
+  message:
+    "More than 10% of the function’s invocations were errors in the selected time range. {{#is_alert}} Resolution: Examine the function’s logs, check for recent code or configuration changes with [Deployment Tracking](https://docs.datadoghq.com/serverless/deployment_tracking), or look for failures across microservices with [distributed tracing](https://docs.datadoghq.com/serverless/distributed_tracing).{{/is_alert}}",
+};
+export const TIMEOUT: ServerlessMonitor = {
+  name: "Timeout on {{functionname.name}} in {{region.name}} for {{aws_account.name}}",
+  query: (cloudFormationStackId: string) => {
+    return `avg(last_15m):sum:aws.lambda.duration.maximum{aws_cloudformation_stack-id:${cloudFormationStackId}} by {aws_account,functionname,region}.as_count() / (sum:aws.lambda.timeout{aws_cloudformation_stack-id:${cloudFormationStackId}} by {aws_account,functionname,region}.as_count() * 1000) >= 1`;
+  },
+  message:
+    "At least one invocation in the selected time range timed out. This occurs when your function runs for longer than the configured timeout or the global Lambda timeout. Resolution: [Distributed tracing](https://docs.datadoghq.com/serverless/distributed_tracing) can help you pinpoint slow requests to APIs and other microservices. You can also consider increasing the timeout of your function. Note that this could affect your AWS bill.",
+};
+export const OUT_OF_MEMORY: ServerlessMonitor = {
```

Comment:
In the README it says you can customize recommended monitors with a custom `threshold` but these values look like they're hard-coded and I don't see any code or tests for changing the threshold for these monitors. Am I missing something here?

Reply:
If this feature is indeed not implemented yet, I would suggest taking it out of the README and implementing it later rather than implementing it in this PR -- like I mentioned in another comment, I think there is already a lot going on in this PR!

Reply:
That's a good point. Every other param except for query (where the critical threshold is defined) can be overridden if the customer has already defined the parameter in their serverless.yml, but I think it would make more sense if we gave customers the ability to override query so that they can tweak parts of it (including the critical threshold) even if it's hard-coded. If that's the case, it should be a pretty straightforward and small change in the code (I'm thinking a few lines). Do you think it would still be worth looking into opening a separate PR?